config/ember-try.js

```diff
@@ -1,74 +1,82 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = async function() {
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-3.4',
+        name: 'ember-lts-3.13',
```

Comment:
didn't readme say 3.24 is min supported version?

Reply:
Good question, I was planning on doctoring up some of this once I could get the workflows running. 3.13 is the first version for Octane, so if we can support back to that I'd like to

Reply:
Huh, this might be a setting that needs to be toggled:
![37cfd5ee4f03845ddb584562ef6ee30c761a52cd_2_690x144](https://user-images.githubusercontent.com/9942917/150426394-9849787e-cc65-4639-b989-31a0ecebe41c.png)

Suppose I could also convert this to a PR off the main repo instead of a fork

Reply:
Where did you find that setting?
I don't see it under https://github.com/kturney/ember-tableish/settings/actions

Reply:
Oh strange -- its showing up in my forked repo's actions, but not within the source repo:
https://github.com/chrismllr/ember-tableish/actions/runs/1725662260

Reply:
Hmm. Maybe like you said an PR from a repo branch instead of a fork may work better